HealthcareSystem/Backend/Repository/RoomRepository/MySqlRoomRepository/MySqlRoomRepository.cs

```diff
@@ -29,7 +29,7 @@ public void DeleteRoom(int number)
 
         public List<Room> GetAllRooms()
         {
-            return _context.Rooms.ToList();
+            return _context.Rooms.Where(d=> !d.Renovation).ToList();
```

Comment:
Does this cause issues with other parts of the system that use this getter? You are essentially using the renovation field for logical deletion, that's kind of weird?? If you want to logically delete rooms rename this field.

Reply:
That room will be created after renovation, so we are unable to use it before the room division.

Reply:
Alright.